src/fontra/views/editor/editor.js

```diff
@@ -2845,6 +2883,57 @@ export class EditorController {
     }
   }
 
+  async glyphSearchDialog(
```

Comment:
Let's call this method `runGlyphSearchDialog()`.

Reply:
done.